### PR TITLE
Space-travellers-hub

### PR DESCRIPTION
**Pull request by megha-n-bodke**
- Create React App.
- implemented navigation bar
- Create routes and view components (Rockets, Missions, My Profile). Use <NavLink 
   /> for the page navigation links and style active class to indicate which section/page user is currently on (underline active 
   navigation link).
- Create directories for all Redux state slice files (rockets, missions).
- Upon first render fetch data from the SpaceX API endpoints:
      - [ ] Rockets: https://api.spacexdata.com/v3/rockets
- Once the data are fetched, dispatch an action to store the selected data in the Redux store.
- Use useSelector() Redux Hook to select the state slices and render lists of rockets and missions in corresponding routes. 
- User can perform "Reserve Rocket" action.
-  Implemented missions section.
- Fetch data from  https://api.spacexdata.com/v3/missions API and update the Redux store.
- Implement functionality that allows users to join the selected mission or leave the mission the user joined earlier.
- Implement myprofile section for reserved mission display.

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import profileReducer from './profile/profile';
+import rocketsReducer from './rockets/rockets';
+import missionsReducer from './Missions/missions';
+
```
Comment (MeqdamAlqudah) [inline src/redux/configureStore.js]: - [ ] Please make sure to use redux logger in your application this is an example of how  to use it 
image

File: src/redux/configureStore.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import profileReducer from './profile/profile';
```
Comment (MeqdamAlqudah) [inline src/redux/configureStore.js]: -  [ ] Please make sure to solve this warning 
image
 
you can use this npm library 
image
you can use it like this 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @megha-n-bodke @SaeqaSultani , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:+1: Readme file
:+1: React files structure


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by medaminedev66 — APPROVED** [review]
Hi @megha-n-bodke and @SaeqaSultani 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 

### Highlights
- Your web app works as expected and runs without errors.
- Highly organized/clean code.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**